provision/initramfs/Makefile.am

```diff
@@ -203,6 +205,8 @@ rootfs: busybox e2fsprogs xfsprogs libarchive parted curl
 	ln -s transporthandler rootfs/sbin/wwgetscript
 	ln -s transporthandler rootfs/sbin/wwgetnodeconfig
 	ln -s transporthandler rootfs/sbin/wwgetfiles
+	rm -f rootfs/usr/bin
```

Comment:
If these are the actual location , why would you try removing them?

Reply:
Right. Something got lost from my original patch in rebasing. Let me push it again ...